src/dom.js

```diff
@@ -284,7 +284,7 @@ class dom {
 		var limit = event.currentTarget.parentNode;
 		var defFns = [];
 
-		while (currElement && currElement !== limit && !event.stopped) {
+		while (currElement && currElement !== limit && !event.stopped && !currElement.disabled) {
```

Comment:
I believe that for the browsers we support all disabled form fields ignore the `click` event. Not sure if all events are ignored though, and I believe this doesn't happen for all elements (like **divs**).

It would be best if you could try to investigate and test the following:
(1) Confirm that all supported browsers (the ones we run tests for in saucelabs, you can see the browser matrix in this repo's READMe.md) work like this.
(2) Check if more events besides **click** are ignored, or if it's just **click**.
(3) Check which elements have this special behavior with **disabled**, probably only form fields (like **button**, **input** and others). I only know of **button** so far.

Once that's known I think this check just needs to be improved to cover all that (probably checking the element's tab and event name, unless it really happens to all elements and events). 

What we want is for delegated events to behave in the same way that native events do, as that's what developers will expect. They're already mostly doing that, but this feature is still missing. People can go around it today, but they shouldn't have to.